scripts/pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "GitPython",
     "PyGitHub",
     "PyYAML",
+    "pre-commit"
```

Comment:
???

Reply:
no, it's required for the pre-commit autoupdate triggered by cruft when instantiating the template

Reply:
OK! I’ll add a comment to clarify that